README.md

```diff
@@ -45,26 +46,38 @@ Currently supported K8S versions are:
 - 1.25
 - 1.24
 
-## What does not work
+### Community Providers
+
+Some cloud providers implemented in machine-controller have been graciously contributed by community members. Those cloud providers are not part of the automated end-to-end
```

Comment:
Do we want to say that those cloud providers are not part or that they might not be part of the automated end-to-end tests run?